cmd/tendermint/commands/light.go

```diff
@@ -222,6 +217,11 @@ func runProxy(cmd *cobra.Command, args []string) error {
 		cfg.WriteTimeout = config.RPC.TimeoutBroadcastTxCommit + 1*time.Second
 	}
 
+	rpcClient, err := rpchttp.NewWithTimeout(primaryAddr, "/websocket", cfg.WriteTimeout)
```

Comment:
This is always some default value, right? What is that value?

Reply:
10 seconds